tests/s3_serializer_test.py

```diff
@@ -108,7 +108,7 @@ def test_dumps_from_s3_serde_min_size_value_error(s3_serializer_factory, s3_clie
         with pytest.raises(S3UploadException) as e:
             data_bytes = b"this is my data"
             s3_uri = s3_serializer_min_size.dumps(data_bytes)
-            assert e.value == "The S3 client couldn't upload the data to the s3 bucket"
+        assert str(e.value) == "The S3 client couldn't upload the data to the s3 bucket"
```

Comment:
is it correct that you removed the indentation?

Reply:
yeap, it was a mistake before! the identation should not be there